package.json

```diff
@@ -47,6 +47,7 @@
     "@babel/preset-typescript": "^7.12.1",
     "@docusaurus/module-type-aliases": "^2.0.0-beta.4",
     "@tsconfig/docusaurus": "^1.0.2",
+    "@types/cheerio": "^0.22.31",
```

Comment:
I noticed there are a lot of changes in`yarn.lock` while there is only a type module changed in `package.json`, I think we should make this PR be focused, do you mind to revert `yarn.lock` and just run `yarn` again but with no upgrades or something?

Reply:
Hi @weareoutman, thanks for a review!

Without updating `yarn.lock` I'm seeing the following issue: `[ERROR] Error: Cannot mix different versions of joi schemas` (with docusaurus `^2.0.0-beta.18`). I've been banging my head quite a lot over this, and the only way I could fix it was by updating the transitive dependencies (via yarn.lock). I suggest we update it, this way we could (1) resolve the "joi schemas" issue and (2) get the latest fixes from the transitive dependencies, which would potentially make the code optimized/secured. What do you think @weareoutman ?

Reply:
Uh, are you using `yarn link`? This is a known issue, see [a workaround here](https://github.com/easyops-cn/docusaurus-search-local/issues/146#issuecomment-1050576012).

I cannot figure out a better way to fix this issue currently.

By the way, there is [a test failed](https://github.com/easyops-cn/docusaurus-search-local/runs/6058901139?check_suite_focus=true) for this PR.

Reply:
Will look into it a bit later, thanks !

Reply:
Sorry this took so long, couldn't get around it as I've been travelling a bit.

The workaround forked just fine, thanks for pointing it out! Now the tests seem not to be failing, the PR seems to be ready for the merge 🎉